tests/native_type_test.fbs

```diff
@@ -0,0 +1,18 @@
+native_include "vector3d.h";
+native_include "vector3d_pack.h";
+
+namespace Geometry;
+
+struct Vector3D (native_type:"Native::Vector3D")
+{
```

Comment:
please follow the same code style as other schemas, meaning `{` on the previous line.

Reply:
Done